onlineShop.core/Managers/CartsManager.cs

```diff
@@ -0,0 +1,178 @@
+﻿using onlineShop.core.Entities;
+using onlineShop.Data;
+using onlineShop.Data.Entities;
+using onlineShop.Managers;
+using onlineShop.Products;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace onlineShop.core.Managers
+{
+    public class CartsManager
+    {
+        private IReservationsProvider _reservationsProvider;
+        private IProductsProvider _productsProvider;
+        private IStocksProvider _stocksProvider;
+        private ReservationsManager _reservationsManager;
+        private ICartsProvider _cartsProvider;
+
+        public CartsManager(IStocksProvider stocksProvider, IProductsProvider productsProvider, IReservationsProvider reservationsProvider, ICartsProvider cartProvider)
```

Comment:
This line is too long, break it into multiple lines, one parameter per line

Reply:
Done